client/ayon_substancedesigner/api/project_creation.py

```diff
@@ -269,3 +348,24 @@ def set_output_resolution_by_graphs(resolution_size_by_graphs):
         graph.setPropertyValue(
             output_size, SDValueInt2.sNew(int2(res_size, res_size))
         )
+
+
+def get_entity_data(project_name):
```

Comment:
Why are we still passing the current project name if _everything_ here is basically aimed at just getting entities for the current context?

Just use: `get_current_context()` function to get project name, folder path and task name in one go.

Also, this may be better named `get_current_context_entities()` or alike, so that it's clear it returns entities.
Additionally, I think this function should be private `_get_current_context_entities()` because it's merely a 'helper' inside this file and shouldn't really be imported elsewhere.